docs/source/researchers-manage-studies.rst

```diff
@@ -73,74 +73,79 @@ Researchers will receive email notifications when their study is approved or whe
+:rejected: Changes have been requested by a CHS admin before the study can be approved.  The study should be edited before resubmitting.
+:active: Study is active and collecting data; participants can access it at the study link. If the study is also marked "Discoverable", the study will show up on CHS's study list.
+:paused: Study is not actively collecting data or visible on CHS; participants cannot access it.
 :deactivated: Study is done collecting data
 :archived: Study has been archived and removed from search
 
-----------------------------------------------------------------------
+===============================================
 Starting and stopping data collection
-----------------------------------------------------------------------
+===============================================
 
 Starting and pausing data collection can be done instantly at any time after your study is approved, using the same dropdown menu as for submitting your study.
 
-What does "starting" your study do? If your study is set as "discoverable" (one of the checkboxes under "edit study"), starting will add your study to the set of studies displayed at `<https://lookit.mit.edu/studies/>`_, and anyone (including you) will be able to participate in it from there. If your study is set as non-discoverable, anyone will be able to participate via a direct link (shown on your study page in the experimenter interface). This is useful for studies intended for a very specific population, for instance if you're doing an online follow-up to an in-person study: you can email the direct link to families, without worrying about screening out other families on Lookit.
+What does "starting" your study do? If your study is set as "discoverable" (one of the checkboxes under "edit study"), starting will add your study to the set of studies displayed at `<https://childrenhelpingscience.com/studies/>`_, and anyone (including you) will be able to participate in it from there. If your study is set as non-discoverable, anyone will be able to participate via a direct link (shown on your study page in the experimenter interface). This is useful for studies intended for a very specific population, for instance if you're doing an online follow-up to an in-person study: you can email the direct link to families, without worrying about screening out other families on CHS.
 
---------------------
-Study edit page
---------------------
-On the study edit page, you can update much of the metadata about the study. You can only view this page if you have permission to view the study details, meaning that you have been given a role specifically on this study OR you are a lab member. If you do not have permission to write study details, you will not be able to make any changes from this page. For more detail about the fields you can view and edit on this page, see "Setting study details."
+===============================================
+Study ad page
+===============================================
```

Comment:
The old "study edit" page is now two pages: "study ad" and "study design". I've moved things into different sections accordingly.

Reply:
Thank you!!